pkg/util/testdata/private_key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Same here

Reply:
This is test data (already being used by apic/auth package)